### DISCUSSION TITLE
BPW34 Selection

### DISCUSSION THREAD

**Discussion by rdagger** [ozel/DIY_particle_detector#38] [Q&A] [answered]
There are several variants of the BPW34.  I see your [Kitspace parts list](https://kitspace.org/boards/github.com/ozel/diy_particle_detector/electron-detector/) recommends the F or FA versions.
I have found inexpensive sources but they are either unmarked or the S version.
Is there a substantial difference between the versions?
![BPW34 Compare](https://user-images.githubusercontent.com/106355/122258531-fae1b400-ce85-11eb-99d3-b81a023a75e4.PNG)

**Comment by ozel** [comment, marked as answer]
you can use any of the diodes in principle. The semiconducting chip is virtually the same (so it seems, see my paper about that :) as far as the ionization effects caused by radioactivity are concerned.
The black cases (F and FA, intended as light filters) help with reducing residual light issues but the clear, unmarked one works too, if the _metal_ enclosure of your detector has no slits between the lid and the rest of the case or some lose hinges that leak light. 
Be careful with the S version, it is wrongly listed as 'through hole' part in your table and the picture is wrong, too. The BPW34 datasheet from Vishay doesn't specify anything about the S version, that's probably the reason for this mismatch. Osram has a separate datasheet which shows the SMD pads: https://de.wikiversity.orghttps://www.osram.com/ecat/DIL%20SMT%20BPW%2034%20S/com/en/class_pim_web_catalog_103489/prd_pim_device_2219543/
If the diodes are manufactured by Osram or Vishay also doesn't matter, by the way (cf. my paper).

If you're adventurous with soldering, you can probably mount the S/SMD version onto the regular footprint for the through-hole version after some careful bending of the pads. But I would not suggest doing that with many diodes, even if they are cheaper. You can of course change the footprint in the KiCad PCB layout file that is also in this repo :-).

In short: If you think your enclosure is well blocking/absorbing ***any environmental light*** completely from/before the diodes, go for the plain BPW34 if it is cheaper for you.

**Reply by rdagger** [reply]
Thanks so much for the quick feedback!
